src/backend/access/zedstore/zedstore_tuplebuffer.c

```diff
@@ -19,17 +19,18 @@
 #include "utils/datum.h"
 #include "utils/hashutils.h"
 
-#define TID_RESERVATION_SIZE		100
+#define SINGLE_INSERT_TID_RESERVATION_SIZE		100
+#define MULTI_INSERT_TID_RESERVATION_FACTOR		10
```

Comment:
Having comment explaining these two would be good.

Reply:
Will add some!